esp8266/README.md

```diff
@@ -69,6 +69,32 @@ $ make PORT=/dev/ttyUSB0 deploy
 
 The image produced is `firmware-combined.bin`, to be flashed at 0x00000.
 
+First start
+-----------
+
+__Serial prompt__
+
+You can access the REPL (Python prompt) over UART resp. USB. 
```

Comment:
respectively USB... we both are old enough to know that, in this case, usb is just a bridge to serial, but many don't see a serial port behind a physical usb port. I wanted to avoid that people get confused from where this serial port is suddenly coming.